src/lib/y2network/clients/auto.rb

```diff
@@ -77,6 +77,8 @@ def change
       end
 
       def import(profile)
+        Yast::NetworkAutoYast.instance.ay_networking_section = profile
```

Comment:
```suggestion
        Yast::NetworkAutoYast.instance.ay_networking_section = 
          AutoinstProfile::NetworkingSection.new_from_hashes(profile)    
```

Reply:
well, does networking section act like Hash? Otherwise I will have to adapt also rest of code.

Reply:
Not really, and we probably use some defaults already, but then we have to document it very well to not be confused as we could expect an `Y2Network::AutoinstProfile::NetworkingSection` object instead of hash and vice versa.